packages/blockchain-utils-validation/src/blockchains/__tests__/tezos.test.ts

```diff
@@ -0,0 +1,183 @@
+import { TezosAuthProvider, TezosProvider } from '../../../../blockchain-utils-linking/src/tezos' //'@ceramicnetwork/blockchain-utils-linking'
```

Comment:
@Geo25rey What is that?!?

Reply:
For some reason tests, fail to resolve local changes to packages, so this is a work around

Reply:
This will break, plz change back and run `npm i` and `npm run build` top level!

Reply:
Sure thing.